build-system/tasks/css/jsify-css.js

```diff
@@ -72,7 +72,12 @@ function transformCss(cssStr, opt_cssnano, opt_filename) {
   );
   const cssnanoTransformer = cssnano({preset: ['default', cssnanoOptions]});
   const transformers = [postcssImport, cssprefixer, cssnanoTransformer];
-  return postcss(transformers).process(cssStr, {
+  /**
+   * TypeScript doesn't like default exports.
+   * @type {*}
+   */
```

Comment:
Possible to fix the `require` statement instead?

Reply:
Fixed